content/tutorials/manage-the-rippled-server/configuration/run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,310 @@
+		}
+	]
+}
+
+
+2021-Dec-09 21:31:53.291502 UTC ReportingETL:NFO Starting reporting etl
+2021-Dec-09 21:31:53.291605 UTC Application:NFO Application starting. Version is 1.8.1+DEBUG
+2021-Dec-09 21:31:53.291747 UTC LoadManager:DBG Starting
+2021-Dec-09 21:31:53.291846 UTC gRPC Server:NFO Starting gRPC server at 0.0.0.0:60051
+2021-Dec-09 21:31:53.293246 UTC LedgerCleaner:DBG Started
+2021-Dec-09 21:31:53.295543 UTC ReportingETL::ETLSource:DBG handleMessage : Received a message on ledger  subscription stream. Message : {
+   "result" : {},
+   "status" : "success",
+   "type" : "response"
+}
+ - { validated_ledger :  , ip : 127.0.0.1 , web socket port : 6006, grpc port : 50051 }
+2021-Dec-09 21:31:53.368075 UTC ReportingETL:NFO monitor : Database is empty. Will download a ledger from the network.
+2021-Dec-09 21:31:53.368183 UTC ReportingETL:NFO monitor : Waiting for next ledger to be validated by network...
+```
+
+## Frequently Asked Questions
+
+### Do I need to run more than one instance of rippled in order to use Reporting mode?
+
+Yes. A rippled node running in reporting mode does not connect to the P2P network, but instead extracts validated data from one or more rippled nodes that are connected to the P2P network. In addition to the rippled node running in reporting mode, you need to run at least one other rippled node that is connected to the P2P network. 
+
+### I’ve already installed rippled. Can I update the configuration file to enable reporting mode and restart rippled? 
+
+Currently, you need to download the source and build rippled for reporting mode. 
+There are initiatives in progress to provide packages for reporting mode.
+
+
+### To run rippled in Reporting mode, I need at least one rippled node running in P2P mode in addition to rippled in Reporting mode. Does this mean I need double the disk space? 
+
+The answer depends on the location of your primary datastore. If you use Cassandra as the primary datastore, the disk space requirement to run rippled in reporting mode will be lower as the data is not stored on your local disk. The PostgreSQL server can be remote as well.
+
+Lastly, the p2p mode server only needs to keep very recent history, since reporting mode will persist long term history.
+
+For more information on system requirements to run rippled, see https://xrpl.org/system-requirements.html. 
+
+### How can I confirm the validity of the data that comes from the Postgres or Cassandra database?
+
+Rippled running in Reporting mode only serves validated data from the ETL source specified in the config file. If you are accessing someone else's rippled server in P2P mode as the ETL source, you are implicitly trusting that server. If not, you need to run your own rippled node in P2P mode.
```

Comment:
Doesn't this also mean that someone could edit the data in Cassandra or PostgreSQL and reporting mode would not know that it has been tampered with? (Question for the SME)